backend/core/src/auth/dto/request-mfa-code.dto.ts

```diff
@@ -0,0 +1,25 @@
+import { Expose } from "class-transformer"
+import { IsEmail, IsEnum, IsOptional, IsPhoneNumber, IsString } from "class-validator"
+import { ValidationsGroupsEnum } from "../../shared/types/validations-groups-enum"
+import { EnforceLowerCase } from "../../shared/decorators/enforceLowerCase.decorator"
+import { MfaType } from "../types/mfa-type"
+
+export class RequestMfaCodeDto {
+  @Expose()
+  @IsEmail({}, { groups: [ValidationsGroupsEnum.default] })
+  @EnforceLowerCase()
+  email: string
+
+  @Expose()
+  @IsString({ groups: [ValidationsGroupsEnum.default] })
+  password: string
+
+  @Expose()
+  @IsEnum(MfaType, { groups: [ValidationsGroupsEnum.default] })
+  mfaType: MfaType
+
+  @Expose()
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
+  @IsPhoneNumber(null, { groups: [ValidationsGroupsEnum.default] })
```

Comment:
This seems to conflict with how twillio handles phone number formats for us domestic numbers. I think we should probably remove it